Formula/x16-emulator.rb

```diff
@@ -0,0 +1,55 @@
+class X16Emulator < Formula
+  desc "Emulator for the Commander X16 8-bit computer"
+  homepage "https://github.com/commanderx16/x16-emulator"
+  url "https://github.com/commanderx16/x16-emulator/archive/r37.tar.gz"
+  sha256 "0c9dbf76ceb5668c32d3b9f04538cdfe891826eaac6bf57c68b13a0152ce15d2"
+  license "BSD-2-Clause"
+
+  depends_on "cc65" => :build
+  depends_on "sdl2"
+
+  resource "rom" do
+    url "https://github.com/commanderx16/x16-rom/archive/r37.tar.gz"
```

Comment:
I just realized that this has a [proprietary license](https://github.com/commanderx16/x16-rom/blob/11f5e7a9e0d48f28abdf52090a12299e061cb1ab/README.md).

What do?

Reply:
We can't package it in homebrew/core then. If this formula can build without this resource then we should remove it. Otherwise the whole formula can't be accepted.

Reply:
Yeah it can build without it, we'll just have to stick with only the help message for `test` (the current test fails on CI anyways, maybe because it creates an SDL GUI)

Reply:
Would the program be usable without this file? (since you're installing it in the install block)
And if it is still usable, would the program fail with an error if you don't specify this file?